force-app/main/default/lwc/geFormWidgetSoftCredit/geFormWidgetSoftCredit.js

```diff
@@ -25,47 +25,22 @@ export default class GeFormWidgetSoftCredit extends LightningElement {
         });
     }
 
-    reset() {
-        // handle reset
-    }
-
-    validate() {
-        // validate
-    }
-
-    get hasAlert() {
-        return false;
+    get softCredits() {
+        if (this.giftInView?.softCredits) {
+            return JSON.parse(this.giftInView.softCredits);
```

Comment:
👍  - I'm glad this works as well as it does performance-wise but look forward to when we don't have to work around it.

Reply:
1000% agree